apps/schedules/lib/parser.ex

```diff
@@ -104,6 +108,19 @@ defmodule Schedules.Parser do
     id
   end
 
+  defp arrival_time(%JsonApi.Item{attributes: %{"arrival_time" => time}}) when not is_nil(time) do
+    Timex.parse!(time, "{ISO:Extended}")
+  end
+
+  defp arrival_time(_), do: nil
+
+  defp departure_time(%JsonApi.Item{attributes: %{"departure_time" => time}})
+       when not is_nil(time) do
```

Comment:
suggestion: could this be `when time do` or `time != nil`? To me `not is_nil` is a little confusing

Reply:
yeah I can do that - added